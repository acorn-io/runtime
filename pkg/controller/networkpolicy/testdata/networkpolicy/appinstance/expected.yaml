---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: app-name
  namespace: app-created-namespace
  labels:
    "acorn.io/managed": "true"
spec:
  ingress:
    - from:
        - namespaceSelector:
            matchLabels:
              acorn.io/app-namespace: app-namespace
  egress:
    - to:
        - ipBlock:
            cidr: 0.0.0.0/0
            except:
              - 169.254.169.254/32
        - ipBlock:
            cidr: ::/0
            except:
              - fd00:ec2::254/64
  podSelector:
    matchLabels:
      acorn.io/app-name: app-name
      acorn.io/app-namespace: app-namespace
      acorn.io/managed: "true"
  policyTypes:
    - Ingress
    - Egress
