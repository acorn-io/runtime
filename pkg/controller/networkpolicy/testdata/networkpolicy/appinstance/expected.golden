`apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  creationTimestamp: null
  labels:
    acorn.io/managed: "true"
  name: app-name
  namespace: app-created-namespace
spec:
  ingress:
  - from:
    - namespaceSelector:
        matchLabels:
          acorn.io/app-namespace: app-namespace
  podSelector:
    matchLabels:
      acorn.io/app-name: app-name
      acorn.io/app-namespace: app-namespace
      acorn.io/managed: "true"
  policyTypes:
  - Ingress
status: {}
`
